lapixdl/formats/lapix.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import annotations
+
+from typing import Any
+
+import pandas as pd
+import shapely.wkt
+
+from lapixdl.base import FileTypeError
+
+
+class Lapix(pd.DataFrame):
+    _metadata = ['geometry', 'category']
+
+    @property
+    def _constructor(self):
+        return Lapix
+
+
+def load(
```

Comment:
Why are the `load`(static) and `save` methods outside the Lapix class?

Reply:
I think the `LapixDataFrame` will be used just inside the library, so nobody will explicitly use it.

 Thinking on this, for me, the code will be cleaner like this:
```python
from lapixdl.formats import lapix

lapix.load(...)
```

That this way:
```python
from lapixdl.formats.lapix import LapixDataFrame

LapixDataFrame.load(...)

# or 

from lapixdl.formats import lapix


lapix.LapixDataFrame.load(...)
```

Also, at ccagt-utils I start using as methods, and I ended up thinking that this brings an extra level of paradigm without any benefit

Reply:
when you read it alone, `lapix.load(...)` does not tell you what is being loaded, while `LapixDataFrame.load(...)` does. Besides, you can import the `load` function alone and use `load(...)`, which makes it even more "obscure" what is happening...